src/overlays/actors/ovl_En_Sda/z_en_sda.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_en_sda.c
  * Overlay: ovl_En_Sda
- * Description:
+ * Description: Dynamic Shadow for Link
```

Comment:
```suggestion
 * Description: Dynamic Shadow for Player
```
afaik we avoid `Link` in the repo in preference for `Player`